packages/resource/src/collection.ts

```diff
@@ -38,3 +36,15 @@ export const resources = <T, Rel extends string>(
   res
     .map(r => (r.payload.length > 0 ? results<T, Rel>(r) : empty<T, Rel>()))
     .getOrElseValue(none<T, Rel>());
+
+export const hasUrl = <T, Rel extends string>(url: string, rel: Rel) => (
+  r: Resource<T, Rel>
+): boolean => {
+  return r.links.reduce<boolean>(
+    (found, l) => (found ? found : l.href === url && l.rel === rel),
+    false
+  );
+};
+
+export const relUrl = <T, Rel extends string>(r: Resource<T, Rel>, rel: string) =>
```

Comment:
Why isn't this in the types.ts with the other URL finders?

Reply:
Fixed